src/Areas/LoadTesting/Services/ILoadTestingService.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using AzureMcp.Areas.LoadTesting.Models.LoadTest;
+using AzureMcp.Areas.LoadTesting.Models.LoadTestingResource;
+using AzureMcp.Areas.LoadTesting.Models.LoadTestRun;
+using AzureMcp.Options;
+
+namespace AzureMcp.Areas.LoadTesting.Services;
+
+public interface ILoadTestingService
+{
+    Task<TestResource> CreateOrUpdateLoadTestingResourceAsync(string subscriptionId, string resourceGroup, string? testResourceName = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null);
+    Task<List<TestResource>> GetLoadTestResourcesAsync(string subscriptionId, string? resourceGroup = null, string? testResourceName = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null);
+    Task<TestRun> GetLoadTestRunAsync(string subscriptionId, string testResourceName, string testRunId, string? resourceGroup = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null);
+    Task<TestRun> CreateOrUpdateLoadTestRunAsync(string subscriptionId, string testResourceName, string testId, string? testRunId = null, string? oldTestRunId = null, string? resourceGroup = null, string? tenant = null, string? displayName = null, string? description = null, bool? debugMode = false, RetryPolicyOptions? retryPolicy = null);
+    Task<List<TestRun>> GetLoadTestRunsFromTestIdAsync(string subscriptionId, string testResourceName, string testId, string? resourceGroup = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null);
+    Task<Test> GetTestAsync(string subscriptionId, string testResourceName, string testId, string? resourceGroup = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null);
+    Task<Test> CreateTestAsync(string subscriptionId, string testResourceName, string testId, string? resourceGroup = null,
+        string? displayName = null, string? description = null,
+        int? duration = 20, int? virtualUsers = 50, int? rampUpTime = 1, string? endpointUrl = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null);
```

Comment:
Update to (string subscription) instead of (string subscriptionId) throughout the codebase.  We're working on updating others to be consistent.